code/src/components/Direction.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { continueLabyrinth } from "reducers/Labyrinth";
+import { useDispatch } from "react-redux";
+import labyrinth from "reducers/Labyrinth";
+import blip from "sounds/effectTwo.wav"
+
+
+export const Direction = ({ description, direction, setDirection }) => {
+  const dispatch = useDispatch();
+
+
+
+  let sound =  new Audio(blip);
+  const SoundPlay = () => {
+    sound.play();
+  };
+
+
+
+  const onClick = () => {
+    dispatch(labyrinth.actions.setAction(direction));
+    dispatch(continueLabyrinth());
+    SoundPlay()
+  };
+
+
+
+
+
+
+
+  return (
+    <div className="DirictionContainer">
+<p>{description}</p>
+<div className="DirectionBtnContainer">
+
+      <button
+      className="DirectionBtn"
+        value={direction}
+        onChange={(e) => setDirection(e.target.value)}
```

Comment:
As far as we understand, this onChange-function isn't needed and probably doesn't even work (see in Gamepage.js)